### ISSUE TITLE
Extensions inserts `undefined` when saving file into the file

### ISSUE THREAD

**Issue by SirWindfield** [SPYGlassMC/SPYGlass#784]
The file before saving (it is under `data/zerotask/functions/beacon_shulker/private/serialize/layers/1`):
```mcfunction
#> zerotask:beacon_shulker/private/serialize/layers/1

scoreboard players set $zt.x zt.temp 3
```

After saving:
```mcfunction
#> zerotask:beacon_shulker/private/serialize/layers/1
undefined

scoreboard players set $zt.x zt.temp 3
```

Any idea why this is happening?

**Comment by ChenCMD** [comment]
This bug is due to the formatter. Also, this only happens if IMP-Doc is listed.

**Comment by SPGoding** [comment]
Fixed in v3 at badbb68460f4d243019283477643cb1e5169458c. Will be released with the next version.
